### PR TITLE
Update gradle and build tools version and fix compilation errors

### PR DESCRIPTION
**Pull request by khurram18**

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @aselle: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 18 days with no activity and the `awaiting review` label has been applied.

File: tensorflow/lite/java/demo/build.gradle [hunk 1]
```diff
@@ -4,9 +4,10 @@ buildscript {
     repositories {
         google()
         jcenter()
+        google()
```
Comment (jdduke) [inline tensorflow/lite/java/demo/build.gradle]: This is redundant

**Comment by jdduke** [comment]
What compilation errors are you seeing? Can you be more explicit?

**Comment by khurram18** [comment]
Unfortunately I am not able to reproduce the compiler errors

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @aselle, @jdduke: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Pull request closed without merging**